### ISSUE TITLE
RDF Satchels opened but not looted before logging out become unopenable

### ISSUE THREAD

**Issue by eTzmNcbkrng** [azerothcore/azerothcore-wotlk#6326]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [ ] Alliance
- [x] Horde

##### CONTENT PHASE:

- [x] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

Logging out after looking inside a Random Dungeon Satchel causes the satchel to be unopenable.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

Satchel should reward loot

##### STEPS TO REPRODUCE THE PROBLEM:

1. `.additem 51999`
2. Open the Satchel with Auto Loot disabled (may require holding Shift or not, depening on your configuration)
3. DO NOT take the item out of the Satchel
4. Press `Esc`
4. `/logout`
5. Log in
6. Open the Satchel

##### EXTRA NOTES:

I suspect this is a regression from https://github.com/azerothcore/azerothcore-wotlk/pull/6151 but unconfirmed.

I also have two satchels sitting on my ChromieCraft character whcih cannot be opened.
Following video shows myself reproducing this on my own Local AzerothCore.
I think the embedded video may not be working - alternative: https://youtu.be/iw24W7bu6G8

https://user-images.githubusercontent.com/26512802/121771587-541ac080-cbc4-11eb-8b51-ac5068982f55.mp4

```
server debug
AzerothCore rev. 6918fdd9993f 2021-06-08 20:02:09 +0000 (master branch) (Win64, RelWithDebInfo, Static)
Using SSL version: OpenSSL 1.1.1j  16 Feb 2021 (library: OpenSSL 1.1.1j  16 Feb 2021)
Using ACE version: 6.5.10
Using Boost version: 1.74.0
Using MySQL version: 80025
Using CMake version: 3.20.2
Compiled on: Windows
Worldserver listening connections on port 8085
Realmlist (Realm Id: 1) configured in port 8085
VMAPs status: Enabled. LineOfSight: 1, getHeight: 1, indoorCheck: 1
MMAPs status: Enabled
maps directory located in R:/azerothcore-wotlk/env/dist/bin/maps. Total size: 252191207 bytes
vmaps directory located in R:/azerothcore-wotlk/env/dist/bin/vmaps. Total size: 658130721 bytes
mmaps directory located in R:/azerothcore-wotlk/env/dist/bin/mmaps. Total size: 2196365052 bytes
Using enUS DBC Locale as default. All available DBC locales: enUS
Using World DB: ACDB 335.4-dev
Using World DB Revision: 2021_06_08_01
Using Character DB Revision: 2021_06_08_00
Using Auth DB Revision: 2021_05_30_00
```

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/cb1aaec4af13341816b65fea5ccbb4b44ef7a585

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by Azcobu** [comment]
Note the lvl 1-30 Satchel is item 51999, not 5199. 

Tried this on my AC instance and had no problems:

https://user-images.githubusercontent.com/81782124/121772533-0cd90380-cbb5-11eb-99cf-c62f49baa73c.mp4

Have you tried this on a pure vanilla client with no mods running? 

Tested on AC rev. b3e64d010d7e 2021-06-11 20:50:35 +0000

**Comment by tkn963** [comment]
Saying they are unopenable after relogging is very misleading. They show as empty.

Client: No addons
Server: Eluna only
Satchel tested: 51999

Added 10 satchels. 2 empty after relogging.
Added 10 satchels, 3 empty after relogging.
Added 10 satchels, 0 empty after relogging.
Added 10 satchels, 2 empty after relogging.
Added 10 satchels, 1 empty after relogging.
Added 10 satchels, 2 empty after relogging.

I can't believe this didn't show up when I tested that PR. Of course I didn't test this many satchels but still.
